src/js/player.js

```diff
@@ -314,6 +314,11 @@ vjs.Player.prototype.loadTech = function(techName, source){
 
   var techReady = function(){
     this.player_.triggerReady();
+
+    // add vjs-autoplay to hide poster and big-play-button if autoplay:true
+    if (this.options_['autoplay']) {
+      this.addClass('vjs-autoplay');
```

Comment:
1) Because I just reused this comparison from another part of the code....
2) I was thinking we might need to remove it but then thought autoplay would either be set or not set.  I can't think of a use case where autoplay would be initially on but then removed.  If we do need to add/remove then I could move the code to the autoplay getter/setter instead.